bin/strata-cli/src/link.rs

```diff
@@ -0,0 +1,184 @@
+use std::fmt::Display;
+
+use alloy::primitives::{Address as StrataAddress, TxHash};
+use bdk_wallet::bitcoin;
+
+/// A thing that exists or is represented onchain.
+/// Currently, this is either a transaction or an address.
```

Comment:
```suggestion
```
This is bad. If we add more enum variants we need to update the docstring, and highly likely this will not be done.